proposals/description/tip_121b.ts

```diff
@@ -0,0 +1,99 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+
+const tip_121b: TemplatedProposalDescription = {
+  title: 'TIP_121b: Protocol ops and technical cleanup',
+  commands: [
+    // 1. Cleanup LUSD->DAI Auction contracts (send dust to TC multisig for selling)
+    {
+      target: 'lusdToDaiSwapper',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Move all DAI and LUSD to daiHoldingPCVDeposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.daiHoldingPCVDeposit, // pcvDeposit
+        addresses.lusd, // token
+        addresses.tribalCouncilSafe, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: 'Move all LUSD from daiHoldingPCVDeposit to the TC Multisig'
```

Comment:
I don't think TC should be doing this, why not execute a sale directly on curve or something?

Reply:
should also include the LUSD in LUSD holding deposit

Reply:
Selling all LUSD on Curve

Reply:
Update: TC is initiating another swap on the LBPs. Will then sell the dust through the TC multisig